discover/portal/articles/07-collaborating/01-publishing-blogs.markdown

```diff
@@ -0,0 +1,444 @@
+
+Its editor delivers features you'd expect, in an elegant in-context manner. It
+has a complete set of WYSIWYG controls that appear where and when you need them.
+You can stay in this easy-to-use mode or switch to source mode to edit your
+content's HTML code. In source mode, you can work with light text on a dark
+background or dark text on a light background. To cap things off, you can do
+fullscreen editing to minimize distractions as your create your posts. You'll
+have a satisfying experience creating awesome blog posts. 
+
+Lastly, Liferay Blogs empowers you to show off your blogs using powerful display
+portlets. You can leverage Liferay's built-in display templates or create your
+own, to present blogs the way you like them. And you can now add a beautiful
+cover image to each of your blog entries. Their like album covers for your
+posts. Let's face it--you might not be able to judge a book by its cover, but
+you're more likely to draw readers to your blog entry by decorating it with an
+enticing cover image! 
+
+By now you're probably chomping at the bit to start blogging, right?! Let's get
+started. 
+
+## Adding Blog Entries [](id=adding-blog-entries)
+
+Each site comes with a built-in blog instance. So let's explore adding an entry
+to your site's blog. The Blogs Administration portlet provides the best place to
+draft blogs. You access it in the Product Menu at *Sites* &rarr; *\[Site Name\]*
+&rarr; *Content* &rarr; *Blogs*. The scope label *Default Scope* shows in
+darkened font below the *Content* header; the scope label indicates you're
+working with the site's blog instance. The *Entries* screen appears, listing the
+site's blog entries. 
+
++$$$
+
+**Note**: To switch to a particular page-scoped blog instance within the site,
+you can click the gear icon and select the page's name. Creating a page-scoped
+blog instance is explained later in the *Scope* portion of the *Displaying
+Blogs* section. 
+
+$$$
+
+Click on the *Add Blog Entry* button to bring up the blog entry editor. 
+
+![Figure 1: This screenshot highlights the blog entry editor's controls for formatting text, justifying images, and editing tables.](../../images/blogs-blogs-entry-mars.png)
+
+The editor's initial screen presents fields to set a cover image, title, and
+subtitle, and an area for creating the entry's content. You may be asking
+yourself, "Where are the editor's controls?" The editor gives you a seamless
+writing experience, displaying controls when you need them, and hiding them from
+view when you don't need them. This keeps your canvas unclutterd, so you can
+focus on writing. As you create content, context-specific controls appear. 
+
+Go ahead and enter text in the *Content* area. If you highlight text, controls
+appear. These controls let you style the text or convert it to a link or a
+tweet--this shares your blog post on Twitter. 
+
+Whenever you park your cursor in the content area, the *Add* icon (`+`)
```

Comment:
Besides clicking the + icon to add images, I think it's also interesting to mention that users can also add images to blog content by dragging a file from desktop and drop it in the blog entry content. At that point, the image will start to be uploaded and will be already part of the blog entry.